kas/repos.py

```diff
@@ -316,10 +316,21 @@ async def apply_patches_async(self):
             elif os.path.isdir(path) \
                     and os.path.isfile(os.path.join(path, 'series')):
                 with open(os.path.join(path, 'series')) as f:
+                    ln = 0
                     for line in f:
-                        if line.startswith('#'):
+                        ln += 1
+                        if not line.rstrip('\n') or line.startswith('#'):
```

Comment:
`rstrip('\n')` accepts empty lines `\n`, but not any whitespace on the line, e.g. ` \n`